Dockerfile

```
@@ -14,12 +14,14 @@ RUN apt-get update && apt-get install -y \
     gdebi \
     #pybedtools dependency
     libz-dev \
-    bedtools=2.25.0-1 \
     #samtools dependencies
     libbz2-dev \
     libncurses5-dev \
     git \
-    python3.7
+    python3.7 \
+    python3.7-dev \
+    libssl-dev \
+    build-essential
```

Maybe we could clear the apt lists too, since we're changing this part of the code already:
```suggestion
    build-essential \
&& rm -rf /var/lib/apt/lists/*
```